### PR TITLE
1920-fixing-allignment-of-effortsheet-and-status-button

### PR DESCRIPTION
**Pull request by chetanbhatt10**
Targets #1920

### Description
the issue in crm->project section that efforts are unalligned for the team members. is solved

### Checklist:

- [x] I have performed a self-review of my own code.

### CODE REVIEW

**Review by GauravGusain98 — CHANGES_REQUESTED** [review]
Please check the comments.

File: Modules/Project/Resources/views/show.blade.php [hunk 1]
```diff
@@ -54,19 +54,20 @@
                     </div>
                     <div class="form-group offset-md-1 pl-4 col-md-5">
                         <h4 class="d-inline-block">
-                            <label for="name" class="font-weight-bold">Status:</label>
+                            <label for="name" class="mt- -2 font-weight-bold">Status:</label>
```
[the remark this answers is not in the document]
Comment (chetanbhatt10) [reply Modules/Project/Resources/views/show.blade.php]: @GauravGusain98 that was a mistake sir, I removed it

**Review by GauravGusain98 — APPROVED** [review]
Looks fine

**Review by GauravGusain98 — APPROVED** [review]
Looks fine

**Pull request merged**